config/settings/testing.py

```diff
@@ -0,0 +1,41 @@
+from config.settings.default import *
+
+DEBUG = True
+
+ALLOWED_HOSTS += ['*']
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
```

Comment:
@ManuelDominic , why do you still have the database engine here as sqlite

Reply:
@kimbugp, i removed it already since i wanted us to use different configurations but then later i changed it since it wasn't necessary.

Reply:
@kimbugp, error updated and please thank you for the feedback.